game/src/assets/assets.json

```diff
@@ -83,5 +83,35 @@
     "js/phaser.min.js",
     "js/game.js",
     "css/main.css"
+  ],
+  "characters": [
+    {
+      "key": "knight_hero",
+      "label": "Valiant Knight"
+    },
+    {
+      "key": "priest_hero",
+      "label": "Avenging Priest"
+    },
+    {
+      "key": "mage_hero",
+      "label": "Magical Mage"
+    },
+    {
+      "key": "fire_monster",
+      "label": "Fire Monster"
+    },
+    {
+      "key": "ice_monster",
+      "label": "Ice Monster"
+    },
+    {
+      "key": "golem_monster",
+      "label": "Golem"
+    },
+    {
+      "key": "spider_hero",
```

Comment:
This should be `spider_monster`.

I think these character definitions should be included as a separate JSON file. See #66 for some ideas.